Makefile

```diff
@@ -182,6 +180,8 @@ db-extras-artix7-harness:
 	+source minitests/roi_harness/arty-swbut.sh && \
 		$(MAKE) -C minitests/roi_harness \
 			HARNESS_DIR=$(XRAY_DATABASE_DIR)/artix7/harness/arty-a7/swbut run copy
+	+XRAY_PIN_00=J13 XRAY_PIN_01=J14 XRAY_PIN_02=K15 XRAY_PIN_03=K16 \
```

Comment:
This moved because there was a race condition in the previous location.  https://github.com/SymbiFlow/prjxray/pull/1227 happens to fix the race condition, so I wouldn't worry about it too much right now.